.gitignore

```diff
@@ -9,6 +9,7 @@ junit.xml
 eslint.xml
 yarn-error.log
 build
+lib64
```

Comment:
This is Python related, not Node. Please move this line under the #Python section, following its sorting and leading slash patterns.